addons/mrp/models/mrp_workorder.py

```diff
@@ -131,6 +132,8 @@ def _read_group_workcenter_id(self, workcenters, domain, order):
     last_working_user_id = fields.One2many('res.users', string='Last user that worked on this work order.', compute='_compute_working_users')
 
     next_work_order_id = fields.Many2one('mrp.workorder', "Next Work Order", check_company=True)
+    previous_work_order_ids = fields.One2many('mrp.workorder', 'next_work_order_id', string="Previous work order", check_company=True)
```

Comment:
previous_work_order_id no? It's a One2many but the relation is always a one to one.